backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/textfield/internal/BpkTextFieldImpl.kt

```diff
@@ -242,6 +252,21 @@ private fun TextFieldBox(
                         else -> BpkTheme.colors.textPrimary
                     },
                 ).value,
+                modifier = Modifier
+                    .padding(end = BpkSpacing.Sm),
```

Comment:
i think this adds unwanted spacing - as you can see the select snapshots were updated and have extra padding now

Reply:
will remove it.